### DISCUSSION TITLE
Open files from `:G diff` view

### DISCUSSION THREAD

**Discussion by svenwltr** [tpope/vim-fugitive#1649] [Q&A] [answered]
(*Context:*) I am a bit confused about the opening of files from withing the `:Git diff master` view. My usual workflow is to do a `git diff master` on my terminal often, when working on a something. It gives my a condensed overview about the current state of my branch. From there I often jump to the files that appear in the diff. It always was kinda cumbersome to do this and jump between terminal and vim.

When I do `:Git diff master`, it gives me the unified diff from my current files to the ones in the `master` branch. From there I am able to directly jump to the file under my cursor and it is even on the correct line. Unfortunately this seems to break some things like autocomplete (from vim-go), `:NERDTreeFind` and occasionally `autocmd` scripts.

I figured out that this seems to be of the file path with looks like `fugitive:///home/me/project/.git//0/my/file.txt` (observed with `CTRL+G`) instead of plain `/home/me/project/my/file.txt`. So my question is whether it is possible to open the file with the plain path?

**Comment by tpope** [comment, marked as answer]
Fugitive is just guessing here; it doesn't understand how to interpret `git diff` arguments and thus doesn't know whether `--cached` was passed, for example. You can give it a bit of extra help by enabling the `diff.mnemonicPrefix` Git configuration option.

**Reply by svenwltr** [reply]
Enabling `diff.mnemonicPrefix` seems to fix my issues. Thank you!
